snap/snapcraft.yaml

```diff
@@ -49,31 +49,39 @@ apps:
 
 parts:      
   rtsp-simple-server:
-    source:
-      - on amd64:
-          https://github.com/aler9/rtsp-simple-server/releases/download/v0.19.2/rtsp-simple-server_v0.19.2_linux_amd64.tar.gz
-      - on arm64:
-          https://github.com/aler9/rtsp-simple-server/releases/download/v0.19.2/rtsp-simple-server_v0.19.2_linux_arm64v8.tar.gz
-    plugin: dump
-    organize:
-      rtsp-simple-server: bin/rtsp-simple-server
-      rtsp-simple-server.yml: config/rtsp-simple-server/config.yml
-    build-packages: [curl]
-    build-snaps: [yq/v4/stable]
+    plugin: nil
+    build-packages: [wget]
+    override-pull: |
+      VERSION=v0.19.2
+
+      ARCH=$(dpkg --print-architecture)      
+      if [ $ARCH == 'arm64' ]; then
+        ARCH=arm64v8
+      fi
+
+      FILE=$(printf 'rtsp-simple-server_%s_linux_%s.tar.gz' "$VERSION" "$ARCH")
+      URL=https://github.com/aler9/rtsp-simple-server/releases/download/$VERSION/$FILE
+
+      # Download and extract archive
+      wget --no-verbose $URL
+      tar -xvf $FILE
+
+      # Download license
+      wget --no-verbose https://github.com/aler9/rtsp-simple-server/raw/main/LICENSE
     override-build: |
-      DOC=$SNAPCRAFT_PART_INSTALL/usr/share/doc/rtsp-simple-server
-      mkdir -p $DOC
-      curl --silent --show-err https://raw.githubusercontent.com/aler9/rtsp-simple-server/main/LICENSE \
-        -o $DOC/LICENSE
-
-      yq -i '
-        .rtmpDisable = "yes" |
-        .hlsDisable = "yes" |
-        .protocols = ["tcp"] |
-        .rtspAddress = "127.0.0.1:8554"
-        ' rtsp-simple-server.yml
-      
-      snapcraftctl build
+      sed --in-place \
```

Comment:
`sed` may silently fail if the upstream configuration changes - https://github.com/edgexfoundry/device-usb-camera/pull/59#pullrequestreview-1049668723

Reply:
That is my comment so I know that :)

As explained in the PR description, the `yq` command hangs in CI builds. We may be able to fix it by finding another yq package. But the cleanest solution I could find was to just replace it with `sed`.